pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -229,6 +260,14 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 		})
 	}
 
+	if r.Plan.Spec.InstallLegacyDrivers || IsLegacyWindows(vm) {
```

Comment:
I want to have also a way how to skip the legacy windows. 
So there could be 3 states of the `InstallLegacyDrivers`
`nil` (default) -> determine by the VM information  
`true` -> use the old drivers 
`false` -> use the new drivers, this is in case something goes wrong with the flow/drivers so we have backup to explicitly use the new drivers